sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -77,10 +77,57 @@ is executed to update some data:
     :source-position: above
 
 
+
+CustomJS for Event classes
```

Comment:
I'd rename this to "CustomJS for User Interaction Events" and rename the above to "CustomJS for Model Property Change Events"